docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,297 @@
+
+In some cases, contributions will require extensive discussions around any
+new APIs, breaking of existing APIs, changing of governance, build, or
+contributing mechanisms, or other aspects of the project. These might
+happen both on the PR itself and on other channels, such as Zulip,
+community meetings, or even one on one discussions. In such situations, the
+PR will not contain sufficient information to document all the
+considerations that went into a decision. Core developers may at their
+discretion then call for a NAP to summarize the discussion to date.
+
+In addition to the above situation, napari is the product of many
+historical decisions that were *not* explained by a NAP. In situations
+where community members might be confused about parts of napari's design,
+and whether alternate designs were considered and rejected, or simply not
+considered. It might then be useful to write a retrospective informational
+NAP to explain that aspect of the project.
+
+### Types of NAPs
+
+There are three kinds of NAPs:
+
+1. A **Standards Track** NAP describes a new feature or implementation
+   for napari.
+2. An **Informational** NAP describes a napari design issue, or provides
+   general guidelines or information to the Python community, but does not
+   propose a new feature. Informational NAPs do not necessarily represent a
+   napari community consensus or recommendation, so users and
+   implementers are free to ignore Informational NAPs.
+3. A **Process** NAP describes a process surrounding napari, or
+   proposes a change to (or an event in) a process. Process NAPs are
+   like Standards Track NAPs but apply to areas other than the napari
+   library itself. They may propose an implementation, but not to
+   napari's codebase; they require community consensus. Examples include
+   procedures, guidelines, changes to the decision-making process, and
+   changes to the tools or environment used in scikit-image development.
+   Any meta-NAP is also considered a Process NAP.
+
+## NAP Workflow
+
+The NAP process begins with a new idea for napari. A NAP should contain a
+single key proposal or new idea. Small enhancements or patches often don't
+need a NAP and can be injected into the napari development workflow with a
+pull request to the napari [repo]. The more focused the NAP, the more
+likely it is to be accepted.
+
+Each NAP must have a champion---someone who writes the NAP using the style
+and format described below, shepherds the discussions in the appropriate
+forums, and attempts to build community consensus around the idea. The NAP
+champion (a.k.a. Author) should first attempt to ascertain whether the idea
+is suitable for a NAP. Posting to the napari [issues list] is the best
+way to do this.
+
+The proposal should be submitted as a draft NAP via a [GitHub pull
+request][github pull request] to the `docs/source/naps` directory with the
+name `nap-<n>-<short-title>.md` where `<n>` is an appropriately assigned
```

Comment:
What is an appropriately assigned number? Any schemes like PEPs have?

Reply:
I've added a clarification that this is "typically sequential", with "typically" indicating that we might deviate from that in some circumstances (though I don't expect to, honestly).